src/io/flutter/run/daemon/FlutterDaemonService.java

```diff
@@ -68,14 +69,16 @@ public void processTerminated(ProcessHandler handler, FlutterDaemonController co
     void deviceRemoved(ConnectedDevice device);
   }
 
-  @Nullable
+  @NotNull
   public static FlutterDaemonService getInstance(@NotNull Project project) {
```

Comment:
Yeah, I think that case is more of a runtime exception - we couldn't construct the class for some reason.